packages/components/src/internal/components/pipeline/PipelineJobsListingPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+
+import { biologicsIsPrimaryApp, sampleManagerIsPrimaryApp } from '../../app/utils';
+
+import { PipelineJobsPage } from './PipelineJobsPage';
+import { PIPELINE_PROVIDER_FILTER_LKB, PIPELINE_PROVIDER_FILTER_LKSM } from './constants';
+
+interface Props {
```

Comment:
Consider using `WithRouterProps` instead. There are other things being passed through besides `params` which we should represent with these typings.

```ts
import { WithRouterProps } from 'react-router';

export const PipelineJobsListingPage: FC<WithRouterProps> = props => {
```